__mocks__/ConsumerMock.ts

```diff
@@ -0,0 +1,9 @@
+export default class ProducerMock {
+	id = 'id';
+
+	resume = jest.fn();
+	pause = jest.fn();
+	setPreferredLayers = jest.fn();
+	setPriority = jest.fn();
```

Comment:
A producer does not have this, and the following method, but doesn't matter

Reply:
Hmmm...producer mock in the wrong file?

Reply:
I think I copied ProducerMock.ts when I created ConsumerMock.ts and forgot to change the class name. The way I import it in mediaMiddleware.test.ts makes it work anyway and I didn't notice. I'll fix it before merging.